wiki/People/The_Team/Beatmap_Nominators/zh.md

```diff
@@ -100,7 +100,7 @@ BN 分为两类：正式 (Full) 成员和见习 (prob) 成员。
 | ::{ flag=US }:: [Yogurtt](https://osu.ppy.sh/users/2649717) |  |
 | ::{ flag=CN }:: [Yugu](https://osu.ppy.sh/users/3161834) | 汉语，日语 |
 | ::{ flag=IT }:: [Zekk](https://osu.ppy.sh/users/9704802) | 意大利语 |
-| ::{ flag=SE }:: [Zer0-](https://osu.ppy.sh/users/4260033) | 瑞典语 |
+| ::{ flag=SE }:: [Kudosu](https://osu.ppy.sh/users/11038155) | 瑞典语 |
```

Comment:
This applies to all languages but shouldnt this be the FR flag? Does he know Swedish or should be French?

Pretty sure Zer0- is still a BN as well, so he should not be removed.

Reply:
Also sort them alphabetically.

Reply:
🤷 tbh the only reason why it was changed is because of ticclick's tool saying that the name was changed. 

I shoulda double checked but for now I won't use the tool.

Reply:
the issue was with wrong identifiers someone copied and pasted, not with the names